pp/integrations/plugin-packs/procedures/network-nokia-timos-snmp.md

```diff
@@ -313,139 +330,149 @@ yum install centreon-plugin-Network-Nokia-Timos-Snmp
+| TRAFFICINBELOWCIR  | Threshold                                                                                                                              |                                                    |             |
+| TRAFFICINABOVECIR  | Threshold                                                                                                                              |                                                    |             |
+| TRAFFICOUTBELOWCIR | Threshold                                                                                                                              |                                                    |             |
+| TRAFFICOUTABOVECIR | Threshold                                                                                                                              |                                                    |             |
+| CRITICALSTATUS     | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %{status}, %{display}               | %{admin\_state} eq "up" and %{oper\_state} !~ /up/ |             |
+| WARNINGSTATUS      | Define the conditions to match for the status to be WARNING. You can use the following variables: %{status}, %{display}                |                                                    |             |
+| EXTRAOPTIONS       | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). | --verbose                                          |             |
+
+</TabItem>
+<TabItem value="Sas-Alarm" label="Sas-Alarm">
+
+| Macro          | Description                                                                                                                                               | Default value                                                                                                                    | Mandatory   |
+|:---------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------|:-----------:|
+| FILTERNAME     | Filter alarm input (tmnxSasAlarmInputDescription) (can be a regexp)                                                                                       |                                                                                                                                  |             |
+| WARNINGSTATUS  | Set warning threshold for status. You can use the following variables: %{alarm\_input\_admin\_state}, %{alarm\_input\_status}, %{alarm\_output\_severity} | %{alarm\_input\_admin\_state} eq "up" and %{alarm\_input\_status} eq "alarm" and %{alarm\_output\_severity} =~ /minor/           |             |
```

Comment:
?

Reply:
Le help dit : 
```
=item B<--warning-status>

Set warning threshold for status. (Default: '%{alarm_input_admin_state} eq "up" and %{alarm_input_status} eq "alarm" and %{alarm_output_severity} =~ /minor/')
You can use the following variables: %{alarm_input_admin_state}, %{alarm_input_status}, %{alarm_output_severity}
```

Donc ça donne : 
Set warning threshold for status. You can use the following variables: %{alarm_input_admin_state}, %{alarm_input_status}, %{alarm_output_severity}

Je comprends pas l'interrogation ^^'